modules/requesting-a-lime-explanation.adoc

```diff
@@ -0,0 +1,13 @@
+:_module-type: CONCEPT
+
+[id='requesting-a-lime-explanation_{context}']
+= Requesting a LIME explanation
+
+[role='_abstract']
+To understand how a model makes its predictions and decisions, you can use a _Local Interpretable Model-agnostic Explanations_ (LIME) explainer. LIME explains a model's decision by showing how much each feature affected the outcome. For example, in a model that decides not to target a user for a marketing campaign, LIME provides a list of weights, both positive and negative, indicating how each feature influenced the model's outcome.
```

Comment:
Regarding:

"For example, in a model that decides not to target a user for a marketing campaign, ..."

Is the model recommending a decision, rather than actually making the decision?